gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatability.kt

```diff
@@ -7,6 +7,7 @@ internal object ComposeCompilerCompatability {
     private val kotlinToCompiler = sortedMapOf(
         "1.7.10" to "1.3.0",
         "1.7.20" to "1.3.2.2",
+        "1.8.0" to "1.4.0-alpha-01",
```

Comment:
According to https://developer.android.com/jetpack/androidx/releases/compose-kotlin#pre-release_kotlin_compatibility 1.4.0-alpha-01 has only Kotlin 1.7.20 compatibility. So shouldn't this be 1.4.0-dev-k1.8.0-33c0ad36f83 instead (see [here](https://androidx.dev/storage/compose-compiler/repository))?

Reply:
1.4.0-alpha-01 is published by jetbrains.  The versions listed here are not published by Google (Jetpack Compose). 
We have them here: https://maven.pkg.jetbrains.space/public/p/compose/dev/org/jetbrains/compose/compiler/compiler/ (Note a different artifact id - org.jetbrains.compose.compiler:compiler, not androidx.compose.compiler:compiler)
And stable versions will be published in mavenCentral later

Reply:
JFYI: Compose Multiplatform 1.3.0-rc02 has been published if you want to give a try

Reply:
@eymar I tried it, but got an error on iOS with compose plugin and serialization plugin: https://github.com/JetBrains/compose-jb/issues/2635